docker/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM hwiese/ubuntu-apt-fast:xenial
```

Comment:
The only thing I see so far. I think we should move this image to a more *professional* place other than my personal Docker account. Besides that at a first glance everything looks fine. I'll test it later today and give my final okay.